src/sqlancer/stonedb/ast/StoneDBJoin.java

```diff
@@ -22,7 +22,7 @@ public static JoinType getRandom() {
     }
 
     public enum NaturalJoinType {
-        FULL, LEFT, RIGHT;
```

Comment:
Why remove FULL?

Reply:
this is not the syntax of MySQL 5.7. I will change `FULL` to `OUTER`